tests/components/homeassistant_yellow/test_init.py

```diff
@@ -305,3 +316,73 @@ async def test_setup_entry_addon_not_running(
 
     assert config_entry.state == ConfigEntryState.SETUP_RETRY
     start_addon.assert_called_once()
+
+
+async def test_create_unseated_repair(hass: HomeAssistant) -> None:
+    """Test that a repair is created when hardware is inconsistent."""
+    mock_integration(hass, MockModule("hassio"))
+
+    # Setup the config entry
+    config_entry = MockConfigEntry(
+        data={},
+        domain=DOMAIN,
+        options={},
+        title="Home Assistant Yellow",
+    )
+    config_entry.add_to_hass(hass)
+
+    with patch(
+        "homeassistant.components.homeassistant_yellow.get_os_info",
+        return_value={"board": "yellow"},
+    ), patch(
+        "homeassistant.components.onboarding.async_is_onboarded", return_value=True
+    ), patch(
+        "homeassistant.components.homeassistant_yellow.check_multi_pan_addon",
+        side_effect=HomeAssistantError(),
+    ), patch(
+        "homeassistant.components.homeassistant_yellow.async_validate_hardware_consistent",
+        return_value=False,
+    ), patch("homeassistant.components.homeassistant_yellow.ir") as ir:
```

Comment:
Don't patch the issue registry. Check that the issue is created instead and assert the issue features.